Cargo.lock

```diff
@@ -1,5 +1,11 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+[[package]]
```

Comment:
Why is Cargo.lock different if nothing in the Toml file changed?

Reply:
I'm also puzzled, but now I did a `cargo update`.

Reply:
Still don't see a Toml update. You should probably just revert the lock file.

Reply:
There is no update to `Cargo.toml`. The practice is to use dependencies like `futures = "0.3.14"` this sets only a minimum requirement for the version. If there is a `0.3.x` version where `x>14` it will be updated. To pin version `futures = "=0.3.14"` has to be used, but the practice is to use first form and let `cargo update` handle the allowed updates for binaries and put the `Cargo.lock` under version control. Side note `cargo install ...` ignores the `Cargo.lock` file, to make it respect `Cargo.lock` `cargo install --locked ...` has to be used.

Reply:
OK that explains it.